### DISCUSSION TITLE
Crab

### DISCUSSION THREAD

**Discussion by Al17er** [creatorsand/.github#112] [Ideas]
**一句话 IDEA：**

sui 链诈骗币清理，通过walrus存储币图标后，项目从walrus获取。

**仓库地址：**
https://github.com/baicaiyihao/crab

**项目展示：**

https://crab.walrus.site/

**我的角色：**

开发&测试

**其他角色需求（不包含发起人）：**
已有项目添加walrus存储功能，团队人员已征齐共3名。

**项目成员**

UI设计: 2n 【 https://github.com/TofuWarrior789 】
开发&测试：stom698 【 https://github.com/baicaiyihao 】
开发&测试：棟梁 【 https://github.com/al17er 】

**社群昵称：**

棟梁、stom698、2n

**个人简介：**

3年安全测试经验，有一定编程基础。

**产品是给谁用的？**

sui链所有用户，用于清理回收链上诈骗币，本次需求添加walrus存储图标功能方便用户更快识别出诈骗币，后续考虑开发api接口供查询。

**其他产品有什么痛点，用户为什么要选择你的产品？**

当前链上暂无同类产品对比，项目中集成了投票环节参与人可对诈骗币进行投票，能够迅速捕获链上诈骗币，通过参与回收用户可获取积分赢取未来潜在空投。

_Originally posted by @ShirleneLiu in https://github.com/CreatorsDAO/.github/discussions/60_

**Comment by baicaiyihao** [comment]
Member:
UI设计: 2n 【[https://github.com/TofuWarrior789】](https://github.com/TofuWarrior789%E3%80%91)
开发&测试：stom698 【[https://github.com/baicaiyihao】](https://github.com/baicaiyihao%E3%80%91)
开发&测试：棟梁 【[https://github.com/al17er】](https://github.com/al17er%E3%80%91)

**Comment by Al17er** [comment]
Project github link: https://github.com/baicaiyihao/crab

**Comment by v1xingyue** [comment]
ai打分

[a comment by an automated account is not shown]
